packages/flutter/lib/src/widgets/tap_and_drag_gestures.dart

```diff
@@ -533,12 +533,16 @@ mixin _TapStatusTrackerMixin on OneSequenceGestureRecognizer {
   // For timing taps.
   Timer? _consecutiveTapTimer;
   Offset? _lastTapOffset;
+  bool _consecutiveTapTimerHasElapsed = false;
 
   // When tracking a tap, the [consecutiveTapCount] is incremented if the given tap
   // falls under the tolerance specifications and reset to 1 if not.
   @override
   void addAllowedPointer(PointerDownEvent event) {
     super.addAllowedPointer(event);
+    if (_consecutiveTapTimerHasElapsed) {
```

Comment:
Could you look at the timer instead to get rid of the boolean?  Something like this:

```dart
if (_consecutiveTapTimer != null && !_consecutiveTapTimer.isActive) {
```

Reply:
That works!